fed/mosaic-mapping/src/main/java/org/eclipse/mosaic/fed/mapping/ambassador/SpawningFramework.java

```diff
@@ -260,6 +283,11 @@ void addVehicleStream(VehicleFlowGenerator vehicleFlowGenerator) {
     }
 
 
+    /**
+     * TODO: unused, keep it?
```

Comment:
Opinion @kschrab ?

Reply:
Oh, no, this should be used.. somewhere the call has been lost during cleanup

Reply:
Re-added.

Reply:
Did you push your changes?

Reply:
Ok, now they're there